modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -82,6 +82,70 @@ image:configuration/files/federation-3.png[image]
 +
 You may disconnect the share at any time by clicking the btn:[trash can] icon.
 
+== Federated Sharing Scanner CronJob Configuration
+
+[IMPORTANT]
+====
+As migration step to 10.5, before enabling this cronjob please remember to remove existing system cron that executes legacy **occ incoming-shares:poll**
+====
+
+Fed Shares Scanner is a background job used to run the federates shares
+scanner over federated shares that are eligible for scanning,
+to ensure integrity of the file cache. Scanner will search for federated shares
+that satisfy these requirements:
+
+. ensure that within single cron run, at max [cronjob_scan_external_batch]
+scans will be performed out of all accepted external shares
+. scan of that external share has not been performed within
+last [cronjob_scan_external_min_scan] seconds
+. user still exists, and has been active recently, meaning logged in at
+least [cronjob_scan_external_min_login] seconds ago
+. federated share root etag/mtime changed, signaling that remote changed
```

Comment:
`federated share root etag/mtime changed, signaling that remote changed and requires scan`

this sentence is not clear to me. how can a local change signal a remote change?

Reply:
It refers to operation of doing request to remote server to check etag/mtime, and if that changed we know we need to do full sync of remote with local

Reply:
I have made a suggestion to make it more clear, see. https://github.com/owncloud/docs/pull/2743#pullrequestreview-468469770